aiida_quantumespresso/common/types.py

```diff
@@ -31,3 +31,12 @@ class SpinType(enum.Enum):
     COLLINEAR = 'collinear'
     NON_COLLINEAR = 'non_collinear'
     SPIN_ORBIT = 'spin_orbit'
+
+
+class RestartType(enum.Enum):
+    """Enumeration of ways to restart a calculation in Quantum ESPRESSO."""
+
+    FROM_SCRATCH = 'from_scratch'
+    FULL = 'full'
+    FROM_CHARGE_DENSITY = 'from_charge_density'
+    FROM_WAVE_FUNCTIONS = 'from_wave_functions'
```

Comment:
Feel free to ignore the following, but looks better aesthetically and order feels more natural as to what they mean. Still I recognize the nitpickiness and subjectiveness degree of this suggestion ^^
```suggestion
    FULL = 'full'
    FROM_SCRATCH = 'from_scratch'
    FROM_CHARGE_DENSITY = 'from_charge_density'
    FROM_WAVE_FUNCTIONS = 'from_wave_functions'
```

Reply:
![image](https://user-images.githubusercontent.com/25607960/134088185-a3a19ab5-fb2a-4f8d-b002-6c72fd2f0d50.png)